website/.gitignore

```diff
@@ -0,0 +1,20 @@
+# dependencies
```

Comment:
a nested gitignore? Is it legit? A good practice?

Reply:
I think it's okay as we're working somewhat like a monorepo here. The website is (except for the docs markdown) a separate project that just lives in the same repo because it makes sense.